.github/ISSUE_TEMPLATE/page-report.yml

```
@@ -0,0 +1,55 @@
+name: "[via MDN pages only]"
+description: Issue filed via "Report a problem with this content on GitHub" link on MDN pages.
+labels: ["via page report", "needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Before you start
+
+        **Want to edit this page?**
+        ↩ Go back and use the "Edit on GitHub" link on the page.
```

```suggestion
        ↩ Go back and use the _Edit on GitHub_ link on the page.
```
to make it similar to the link reference in next line